templates/admin/judge/judge/change_form.html

```diff
@@ -22,5 +23,10 @@
             <i class="fa fa-lg fa-plug"></i>
             <span class="text">{% trans "Terminate" %}</span>
         </a>
+        <a style="display: none" title="{% trans "Disable" %}" href="{% url 'admin:judge_judge_disable' original.pk %}"
+           class="button disable-link">
+            <i class="fa fa-lg fa-ban"></i>
+            <span class="text">{% trans "Disable" %}</span>
```

Comment:
Don't know how I feel about this being a toggle but always showing `Disable`.

Reply:
This has been addressed.